crates/orchestrator/src/jobs/da_job/mod.rs

```diff
@@ -179,4 +178,36 @@ mod tests {
         let expected = FieldElement::from_dec_str(expected.as_str()).unwrap();
         assert_eq!(da_word, expected);
     }
-}
+
+    mod test_state_update_to_blob_data {
+        use super::*;
+        use std::fs::{read_to_string, File};
+        use std::io::{self, BufRead};
+        use serde_json::Error;
+
+        #[test]
+        fn state_update_to_blob_data_works() {
+            let state_update_path = "test-utils/stateUpdate.json".to_owned();
+            let contents = read_to_string(state_update_path).expect("Couldn't find or load that file.");
+
+            let v: Result<StateUpdate, Error> = serde_json::from_str(&contents.as_str());
```

Comment:
you can use .expect here with the error message directly instead of doing a match later on. .expect panics if there's an Err